src/libs/SidebarUtils.ts

```diff
@@ -381,7 +389,7 @@ function getOptionData({
         } else {
             result.alternateText = lastMessageTextFromReport.length > 0 ? lastMessageText : ReportActionsUtils.getLastVisibleMessage(report.reportID, {}, lastAction)?.lastMessageText;
             if (!result.alternateText) {
-                result.alternateText = Localize.translate(preferredLocale, 'report.noActivityYet');
+                result.alternateText = ReportUtils.formatReportLastMessageText(getReportBeginningOfChatHistoryMessage(report)) || Localize.translate(preferredLocale, 'report.noActivityYet');
```

Comment:
The fallback is almost never used. The `getReportBeginningOfChatHistoryMessage` function should return `undefined` for cases where we don't have a "beginning of chat" text